packages/ui-components/src/components/Inputs/MultiSelect.tsx

```diff
@@ -19,6 +19,7 @@ const StyledCheckbox = styled(Checkbox)`
 const MenuItem = styled(MuiMenuItem)`
   padding-top: 0;
   padding-bottom: 0;
+  background-color: white;
```

Comment:
Same here, this gets imported in multiple places and might have unintended consequences.